src/sentry/shared_integrations/client.py

```diff
@@ -70,12 +70,13 @@ def from_response(self, response, allow_text=False):
                     response.headers.get("Content-Type", ""), response.status_code
                 )
         else:
-            data = json.loads(response.text, object_pairs_hook=OrderedDict)
-
+            data = json.loads(response.text or "null", object_pairs_hook=OrderedDict)
         if isinstance(data, dict):
             return MappingApiResponse(data, response.headers, response.status_code)
         elif isinstance(data, (list, tuple)):
             return SequenceApiResponse(data, response.headers, response.status_code)
+        elif data is None:
```

Comment:
Unsure about this change too, but it's because of the other one in this file. I needed to handle the empty string response somehow.